### PR TITLE
imported and exposed support for character order and measurement units and systems

### PR DESCRIPTION
**Pull request by alexbodn**
imported and exposed support for character order:
	>>> Locale('de', 'DE').character_order
	<<< u'left-to-right'
	>>> Locale('ar', 'SA').direction
	<<< u'rtl'

exposed measurement units and measurement systems translation:
	>>> locale = Locale('fr', 'FR')
	>>> val = 12
	>>> unit_length = 'long'
	>>> unit = 'length-foot' + ':' + unit_length
	>>> locale.unit_patterns[unit][locale.plural_form(val)].format(val)
	<<< u'12 pieds'
	>>> locale = Locale('fr', 'FR')
	>>> print locale.measurement_systems['US']
	<<< américain

### CODE REVIEW

**Comment by alexbodn** [comment]
hello friends,

being overwhelmed by the upstream repository changing rate, 
and not being such a proficient committer, 
i have just remade my fork and immediately patched it, 
to be able to request a pull before it will be too late again.

i will soon change the unit_patterns and measurement_systems properties 
to methods with default parameters.

hope this could be pulled,
alex :)

File: babel/core.py [hunk 1]
```diff
@@ -885,6 +885,52 @@ def ordinal_form(self):
...
+        u'12 pieds'
+        """
+        return self._data['unit_patterns']
+
+    @property
+    def measurement_systems(self):
+        """Locale names for the measurement systems.
+
+        >>> locale = Locale('fr', 'FR')
+        >>> print locale.measurement_systems['US']
```
Comment (akx) [inline babel/core.py]: Remove the word `print ` here; the tests are failing because of it.

File: babel/core.py [hunk 2]
```diff
@@ -885,6 +885,52 @@ def ordinal_form(self):
...
+        américain
+        """
+        return self._data['measurement_systems']
+
+    @property
+    def character_order(self):
+        """this locale text direction . the original form.
```
Comment (akx) [inline babel/core.py]: Rephrase this to something like

> The text direction for the locale ("left-to-right" or "right-to-left").

File: babel/core.py [hunk 3]
```diff
@@ -885,6 +885,52 @@ def ordinal_form(self):
...
+        >>> Locale('de', 'DE').character_order
+        u'left-to-right'
+        >>> Locale('ar', 'SA').character_order
+        u'right-to-left'
+        """
+        return self._data['character_order']
+
+    @property
+    def direction(self):
+        """this locale text direction  the css form.
```
Comment (akx) [inline babel/core.py]: Same here;

> The text direction for the locale in CSS form ("ltr" or "rtl").

**Comment by alexbodn** [comment]
done @akx. a big thank.

**Comment by akx** [comment]
@alexbodn: Unfortunately, the tests still fail.

* `unicode` is not a thing on Python 3; the cast really shouldn't be necessary at all though
* On Python 2 (and likely Python 3 too), the return value is `u'am\xe9ricain'`.

**Comment by alexbodn** [comment]
hello @akx, 
a very big thank for bearing with my small work that has so many issues.
fixed your last comment too. just forgot to send the comment :)

**Comment by akx** [comment]
@alexbodn Unfortunately now the tests fail on Python 2 (though they do work on Python 3). Would you like to try and fix that, or would you like me to grab your patches and fix them to work (in another PR)?

**Comment by alexbodn** [comment]
first, my code is yours :) so please feel free to do as you see fit.
i made my fork only to contribute these patches, and would do my best to help.
ironically, i use python 2 only but, as you can see, i don't know how to test.
so, please try and again tell me whether i have to do something on my part :)

**Comment by akx** [comment]
Closing in favor of #368 :)
